src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,62 @@
+@import '../../style/mixin.scss';
+
+* {
+  box-sizing: border-box;
+}
+
+.introduce {
+  width: 470px;
+  height: 690px;
+  margin: 100px auto;
+  border: 2px solid #000000;
+  border-radius: 15px;
+  box-sizing: border-box;
+}
+
+.introduce {
+  .blackBox {
+    @include flexCenter;
+    width: 390px;
+    height: 390px;
+    margin: 40px auto;
+    border-radius: 15px;
+    background-color: #000000;
+    color: #ffffff;
+    font-size: 80px;
+  }
```

introduce라는 부분이 동시에 되어있네요!
introduce 스타일을 적용하려면 아래와 같이 변경할 수 있겠습니다.

```suggestion
.introduce {
  width: 470px;
  height: 690px;
  margin: 100px auto;
  border: 2px solid #000000;
  border-radius: 15px;
  box-sizing: border-box;
  
  .blackBox {
    @include flexCenter;
    width: 390px;
    height: 390px;
    margin: 40px auto;
    border-radius: 15px;
    background-color: #000000;
    color: #ffffff;
    font-size: 80px;
  }
```